src/main/java/core/basesyntax/ReportCreator.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class ReportCreator {
+    public static final int ACTION_INDEX = 0;
+    public static final int COUNT_INDEX = 1;
```

Comment:
which access modifiers you should use here, considering that you use these constants only in this class?